tests/testthat/test-utils.R

```diff
@@ -52,16 +52,17 @@ test_that("padding_lst applies padding to a flextable based on indentation level
   expect_is(padded_ft, "flextable")
 })
 
-block_text <- "Line 1\nLine 2\nLine 3\nLine 4\nLine 5\nLine 6"
+l <- 5
+block_text <- paste(paste(rep("Line", l), seq_len(l)), collapse = "\n")
 
-test_that("split_text_block - splits text block into equal parts", {
+test_that("split_text_block - n < l : splits text block into blocks no longer than n lines", {
```

Comment:
The comment alone is not clear, one must read the code above to know what `l` is. `n` is fine as that is an argument to the function.

Reply:
I will keep the `line_break` instead of `l`. This should be enough as I don't want to clutter the small test with too much comments.

Reply:
But I don't know what `line_break` is either.

Reply:
What do you suggest?  I can retain the original comment.

```
line_break <- 5
block_text <- paste(paste(rep("Line", line_break), seq_len(line_break)), collapse = "\n")

test_that("split_text_block - splits text block into blocks no longer than n lines", {
  n <- 2
  result <- split_text_block(block_text, n)
  result_lines <- lapply(result, function(x) strsplit(x, "\n")[[1]])
  lapply(result_lines, function(x) expect_lte(length(x), n))
})

test_that("split_text_block - returns single block with original text", {
  n <- 7
  result <- split_text_block(block_text, n)
  expect_equal(result, list(block_text))
})
```

Reply:
Yes, just leave it.
And `line_break` makes no sense, btw.